File: backend/src/main/java/hu/szakdoga/backend/forum/data/dto/MajorDTO.java
package hu.szakdoga.backend.forum.data.dto;

public class MajorDTO {
    public final Long id;
    public final String name;
    public final String abbreviation;

    public MajorDTO(Long id, String name, String abbreviation) {
        this.id = id;
        this.name = name;
        this.abbreviation = abbreviation;
    }
}

File: frontend/src/app/shared/model/forum/major.dto.ts
export class MajorDto {
    public readonly id: number | null;
    public readonly name: string;
    public readonly abbreviation: string;


	constructor(name: string, abbreviation: string, id: number | null = null){
        this.id = id;
        this.name = name;
        this.abbreviation = abbreviation;
    }
}